src/core/hook/wee-hook-url.c

```diff
@@ -220,11 +231,25 @@ hook_url_transfer (struct t_hook *hook)
                          &hook_url_transfer_thread, hook);
     if (rc != 0)
     {
-        gui_chat_printf (NULL,
-                         _("%sError running thread in hook_url: %s (URL: \"%s\")"),
-                         gui_chat_prefix[GUI_CHAT_PREFIX_ERROR],
-                         strerror (rc),
-                         HOOK_URL(hook, url));
+        snprintf (str_error, sizeof (str_error),
+                  "error calling pthread_create (%d)", rc);
```

Comment:
Should this be translated? Related to this comment: https://github.com/weechat/weechat/commit/6468f7761f9a202bce29f4f958e783112d751fc8#r129887449

Reply:
I think it should not be translated as it's not for display, but an info sent to the callback (so it's easier to parse in a plugin/script).